scripts/travis/update/update-project.sh

```diff
@@ -1,5 +1,8 @@
 #!/usr/bin/env bash
 
+git config user.name "alfresco-build"
```

Comment:
can we avoid to hardcode it?

Reply:
if you think it's fine to execute those two commands in the before_install travis phase I can author a small config in alfresco-build-tools that can be imported here (like this [maven config](https://github.com/Alfresco/alfresco-build-tools/blob/master/.travis.maven_config.yml)) - we still had hardcode but hopefully in a single place

Reply:
try with https://github.com/Alfresco/alfresco-build-tools/pull/121